### PR TITLE
Fix panel not active message.

### PR DESCRIPTION
**Pull request by Harry-Hopkinson**
For #389

### CODE REVIEW

File: src/extension/extension.ts [hunk 1]
```diff
@@ -187,7 +187,6 @@ export function storeCollectionAsMemento(
     ]);
 }
 
-let petPlaygroundStatusBar: vscode.StatusBarItem;
```
Comment (Harry-Hopkinson) [inline src/extension/extension.ts:190]: This is unused as the spawnPetStatusBar is the one actually displayed

File: src/extension/extension.ts [hunk 2]
```diff
@@ -309,14 +308,6 @@ export function activate(context: vscode.ExtensionContext) {
         }),
     );
 
-    // status bar item
```
Comment (Harry-Hopkinson) [inline src/extension/extension.ts:312]: Can remove this as associated with petPlaygroundStatusBar

**Comment by Harry-Hopkinson** [comment]
@tonybaloney this PR should be ready

**Comment by tonybaloney** [comment]
please remove changes to `package-lock.json` from the branch.

**Comment by Harry-Hopkinson** [comment]
I have removed the package-lock changes.

**Review by tonybaloney — APPROVED** [review]

**Comment by tonybaloney** [comment]
thanks for the PR

**Pull request merged**